d3a_interface/sim_results/kpi.py

```diff
@@ -145,11 +145,74 @@ def update_area_kpi(self, area_dict, core_stats):
         self._accumulate_energy_trace(core_stats)
 
 
+class SavingsKPI:
+    def __init__(self):
+        self.producer_ess_set = set()
+        self.consumer_ess_set = set()
```

Comment:
Why is `ess` part of the name? I got confused and thought that it is a set of storages. But it is not, right?

Reply:
https://github.com/gridsingularity/d3a-interface/pull/223#discussion_r654221854